### PR TITLE
[Documentation] Copied Sylius-Docs to docs/

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | #4563 
| License         | MIT

Documentation copied into `docs` preserving commit history

### CODE REVIEW

**Comment by patkar** [comment]
Awesome, can I ask how you have done it?

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+| Q             | A
```
Comment (lchrusciel) [inline docs/PULL_REQUEST_TEMPLATE.md]: Some of files can be deleted, but because of diff we can wait with that

**Comment by pamil** [comment]
@patkar I had to use StackOverflow Driven Development :tada: In the end I came up with something like this:

```sh
git remote add Sylius-Docs https://github.com/Sylius/Sylius-Docs.git
git fetch Sylius-Docs
git merge -s ours --no-commit Sylius-Docs/master
git read-tree --prefix=docs -u Sylius-Docs/master
git commit -m "Foo bar"
```

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (pamil) [inline docs/PULL_REQUEST_TEMPLATE.md]: :+1:, but in another PR :)

**Comment by pjedrzejewski** [comment]
Good morning. :tada: Thanks Kamil!

**Pull request merged**